src/firefoxprofilerunner.cpp

```diff
@@ -58,17 +58,29 @@ void FirefoxRunner::reloadPluginConfiguration(const QString &configFile) {
         matchActions.clear();
     }
 
-    QList<Plasma::RunnerSyntax> syntaxes;
-    syntaxes.append(Plasma::RunnerSyntax("firefox :q?",
-                                         "Plugin gets triggered by firef... after that you can search the profiles by name")
-    );
-    syntaxes.append(Plasma::RunnerSyntax("firefox :q -p", "Launch profile in private window"));
-    setSyntaxes(syntaxes);
+    setDefaultSyntax(Plasma::RunnerSyntax(
```

Comment:
Please add this as a normal syntax. The default syntax concept is unused and is deprecated.

Reply:
Done.

Reply:
Thanks!